tests/itest_specs/basic_round_trip/long_call.cairo

```diff
@@ -47,10 +47,14 @@ namespace LongCallRoundTrip {
             ids.opt_long_call_addr = context.opt_long_call_addr_0
         %}
 
+        tempvar tmp_address = EMPIRIC_ORACLE_ADDRESS;
         %{
             stop_prank_amm = start_prank(context.admin_address, context.amm_addr)
-            stop_mock_oracle_1 = mock_call(
-                ids.EMPIRIC_ORACLE_ADDRESS, "get_value", [1400000000000000000000, 18, 0, 0]  # mock current ETH price at 1400
+            stop_mock_current_price = mock_call(
+                ids.tmp_address, "get_spot_median", [1400000000000000000000, 18, 0, 0]  # mock current ETH price at 1400
+            )
+            stop_mock_terminal_price = mock_call(
+                ids.tmp_address, "get_last_checkpoint_before", [0,145000000000, 0, 0, 0]  # mock terminal ETH price at 1450
```

Comment:
omg, they have different number of decimals for get_spot_median and get_last_checkpoint_before... 

I guess that the get_spot_median has it defined by the "second number" which is 18 in this case...  adn the   get_last_checkpoint_before does not have it at all...

Reply:
@Chepelau please do, and let me know when its done :)

Reply:
Done